src/linuxforhealth/csvtofhir/model/csv/basic.py

```diff
@@ -0,0 +1,18 @@
+from typing import List, Optional
+from pydantic import validator
+
+from linuxforhealth.csvtofhir.model.csv.base import CsvBaseModel
+from linuxforhealth.csvtofhir.model.csv.base import BaseModel
+
+
+class IdentifierDetails(BaseModel):
+    token_name: Optional[str]
```

Comment:
Since this class is now used for both token and non-token identifiers, `token_` should be removed from the field names